<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneG4VolumeConfiguration</class>
 <widget class="QDialog" name="OneG4VolumeConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
   <height>407</height>
  </rect>
 </property>
  <property name="windowTitle">
   <string>Volume Control Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device to control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="audioBackendLabel">
          <property name="text">
           <string>Backend:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="audioBackendCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="devAddedCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="muteOnMiddleClickCheckBox">
        <property name="text">
         <string>Mute on middle click</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreMaxVolumeCheckBox">
        <property name="text">
         <string>Allow volume beyond 100% (0dB)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alwaysShowNotificationsCheckBox">
        <property name="text">
         <string>Always notify about volume changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showKeyboardNotificationsCheckBox">
        <property name="text">
         <string>Notify about volume changes with keyboard</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Volume adjust step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="stepSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
    </layout>
   </widget>
  </item>
   <item>
    <widget class="QGroupBox" name="policyGroupBox">
     <property name="title">
      <string>WirePlumber policy (requires restart)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="policyHintLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Disable sinks/devices via WirePlumber rules. Apply and restart to take effect.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="policyTree">
        <column>
         <property name="text">
          <string>Output</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Node name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Disable device</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Disable node</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="policyStatusLabel">
          <property name="text">
           <string>Saved.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="applyPolicyButton">
          <property name="text">
           <string>Apply and restart</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>OneG4VolumeConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>OneG4VolumeConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
